doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -8,11 +8,25 @@ cells:
     LArray provides methods and functions to load and dump LArray, Session, Axis Group objects to several formats such as Excel, CSV and HDF5. The HDF5 file format is designed to store and organize large amounts of data. It allows to read and write data much faster than when working with CSV and Excel files. 
 
 
+- code: |
+    # remove this once the notebook is ready
+    import sys
+    import os
+    sys.path.insert(0, os.path.abspath('../../../'))
+
+  id: 0
+
 - code: |
     # first of all, import the LArray library
     from larray import *
+    
+    # then, let's define a function to get full path of any example file
+    def get_filepath(fname):
```

Comment:
maybe we should add a function like that to larray itself?

Reply:
I don't really think that users will play with example files outside the tutorial.

Reply:
maybe not, but it would make the tutorial and at least one doctest a bit more readable

Reply:
OK. It doesn't hurt anyway.